awx/settings/defaults.py

```diff
@@ -967,6 +967,10 @@
 # K8S only. Use receptor_log_level on AWX spec to set this properly
 RECEPTOR_LOG_LEVEL = 'info'
 
+RECEPTOR_INSTALLATION_METHODS = ['release', 'package', 'local', 'container']
```

Comment:
I don’t see this variable used. Can we delete it?

Reply:
would be used by the UI to populate the selection drop down 

also somewhere we want to be able to limit what options are "exposed" for example downstream "local" and "release" wont be available